internal/commands/fix.go

```diff
@@ -0,0 +1,64 @@
+package commands
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/eikendev/hackenv/internal/options"
+	"github.com/eikendev/hackenv/internal/scripts"
+	log "github.com/sirupsen/logrus"
+)
+
+// FixCommand struct
+type FixCommand struct {
+	CreateBridge createBridge `cmd:"create-bridge" aliases:"c" help:"Create bridge"`
+	RemoveBridge removeBridge `cmd:"remove-bridge" aliases:"r" help:"Remove bridge"`
+	FixLabels    fixLabels    `cmd:"fix-labels" aliases:"l" help:"Fix SElinux labels"`
+	All          all          `cmd:"all" aliases:"a" help:"Create bridge and fix selinux labels"`
+}
+
+type createBridge struct{}
+
+func (c *createBridge) Run(s *options.Options) error {
+	return execCommand([]string{scripts.CreateBridgeScript}, s.Verbose)
+}
+
+type removeBridge struct{}
+
+func (c *removeBridge) Run(s *options.Options) error {
+	return execCommand([]string{scripts.RemoveBridgeScript}, s.Verbose)
+}
+
+type fixLabels struct{}
+
+func (c *fixLabels) Run(s *options.Options) error {
+	return execCommand([]string{scripts.FixLabelsScript}, s.Verbose)
+}
+
+type all struct{}
+
+func (c *all) Run(s *options.Options) error {
+	if err := execCommand([]string{scripts.CreateBridgeScript, scripts.FixLabelsScript}, s.Verbose); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Why is the pattern here different compared to the other `Run()` functions?

Reply:
I'm not sure to understand what you mean here but for sure i got an useless err check here, is it ok now?